YangLu/task01/quicksort/build.gradle

```diff
@@ -14,13 +14,9 @@ repositories {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib"
 
-    implementation 'com.github.inforion:kotlin-logging:v0.2.1'
-    implementation 'com.github.inforion:kotlin-extensions:v0.3.2'
-
-    compile group: 'mysql', name: 'mysql-connector-java', version: '8.0.21'
 
     compileKotlin {
-        kotlinOptions.jvmTarget = "11"
+

```

Comment:
Have you even compiled? This gradle code is incorrect

Reply:
Yes,I did.But this is not my code of last version. It's different from what on my github. I'll try to pull again.

Reply:
![E7E5FBCD-42A8-4adc-9F77-9AB8F3E573A5](https://user-images.githubusercontent.com/55850277/93617287-5de20f80-fa08-11ea-8d00-2555bf98283b.png)
![F01CF073-20EE-41f7-A1E0-65A0CA085B88](https://user-images.githubusercontent.com/55850277/93617313-65091d80-fa08-11ea-92a1-782077120259.png)

Reply:
![6EF617D9-A823-42cf-B49D-59D4FF21FE74](https://user-images.githubusercontent.com/55850277/93617516-a7caf580-fa08-11ea-9aa6-16b5ba7b1f7b.png)

Reply:
![6EF617D9-A823-42cf-B49D-59D4FF21FE74](https://user-images.githubusercontent.com/55850277/93618227-90d8d300-fa09-11ea-9443-63ee59205731.png)